.idea/misc.xml

```diff
@@ -4,5 +4,5 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" project-jdk-name="11" project-jdk-type="JavaSDK" />
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="false" project-jdk-name="11" project-jdk-type="JavaSDK" />
```

Comment:
It again changed to jdk 11. Shouldn't it be jdk 8 for compatibility?

Reply:
Yeah, I'll probably just ignore this file now. I've set it 8 a couple of times now.